client/view/QueryLink/QueryLink.css

```diff
@@ -6,6 +6,11 @@
   &.is-pre {
     border-radius: 8px;
   }
+
+  &.fix {
```

Comment:
Modifier classes has `is-X` pattern

Reply:
Maybe `is-big` is a better name. We should explain what is different, not where it is used.

Reply:
Maybe `is-block`? or you mean `is-big` like take more space?

Reply:
With `This branch has conflicts that must be resolved` I can’t see a preview and understand how this button looks like to recommend you a good name.

Reply:
Pardon, resolved

Reply:
Oops, my script is not ready for third-party PR preview deploy :(.

Can you show me screenshot?

Reply:
ok, few minutes

Reply:
Can you also rebase this commit? https://github.com/browserslist/browsersl.ist/commit/1e50f09395d855bbb6c0dce6ac37c79b26de946a

Reply:
<img width="1439" alt="Screenshot 2022-09-25 at 14 29 41" src="https://user-images.githubusercontent.com/22856389/192139080-46618c90-d469-4b87-9e8b-023077635a51.png">

Reply:
Looks bad:

1. Blue on yellow doesn’t fit. You need to change text color.
2. `display: block` tooks too many vertical space. It is better to move the button to the right of the warning.

Reply:
i could leave here short video link if need?, and i think need to fix that when click on button, button is slide a little to the right, need to check what is this behavior

Reply:
1. Let's rebase branch again. Maybe deploy will work this time and we could see live preview
2. It is better to fix position and color first. Maybe changes in position will fix `:active` behavior.

Reply:
I have one question which color will be more compatible?, i tried orange, and marsianse that have on page, and both not expressed
<img width="590" alt="Screenshot 2022-09-25 at 15 50 39" src="https://user-images.githubusercontent.com/22856389/192142755-48d17242-6080-482f-b924-d073c22181d6.png">
![photo_2022-09-25 15 50 46](https://user-images.githubusercontent.com/22856389/192142761-d04c81eb-02b4-4738-aa1b-7114aa24ff76.jpeg)

Reply:
Second option is a little better, but:
1. Text is orange but button background is transparent blue
2. I think it has low contrast


I see two option:
1. Use transparent black for background and black for text
2. Use darker orange for text and transparent background